clang/lib/CIR/CodeGen/CIRGenBuilder.cpp

```diff
@@ -130,6 +132,44 @@ void CIRGenBuilderTy::computeGlobalViewIndicesFromFlatOffset(
   computeGlobalViewIndicesFromFlatOffset(offset, subType, layout, indices);
 }
 
+static mlir::Type getAttributeType(mlir::Attribute attr) {
+  return mlir::cast<mlir::TypedAttr>(attr).getType();
+}
+
+cir::RecordType clang::CIRGen::CIRGenBuilderTy::getCompleteRecordType(
```

Comment:
I don't love having both `getCompleteRecordTy` and `getCompleteRecordType`. Can we give them the same name and put the common parameters in the same order?